docs/UserGuide.md

```diff
@@ -182,33 +216,33 @@ This section contains commands for managing students.
 
 ### Adding a student: `addstudent`
 
-Adds a student to JARVIS.
+Adds a student to your student list.
 
 Format: `addstudent s/NAME m/MATRIC_NUM`
 
 * `m/MATRIC_NUM` must start with 'A', followed by 7 digits and a capital letter.
-* A maximum of 10 students can exist in the student list at any point in time. The user will not be allowed to add more students after reaching the limit of 10 students.
-* `MATRIC_NUM` provided must not already exist in student list. 
+* A maximum of 10 students can exist in your student list at any point in time. You will not be allowed to add more students after reaching the limit of 10 students.
```

Comment:
If we put this under information_source, I think we would have to do the same for a lot of commands. It might look too cluttered with `Note:`. The ones I added for the `list` commands are more of additional information thats nice to know.